src/transformation/visitors/variable-declaration.ts

```diff
@@ -26,12 +26,16 @@ export function transformArrayBindingElement(
     }
 }
 
-export function transformBindingPattern(
+export function createDestructuringStatements(
```

Comment:
`pattern: ts.BindingPattern` is being transformed inside, so I think it should start with `transform`